site/specs-temp/numbers.json

```diff
@@ -30001,6 +30566,295 @@
+              "$ref": "#/components/schemas/Error"
+            }
+          }
+        }
+      },
+      "ImportVoiceTnOrderError": {
+        "type": "object",
+        "properties": {
+          "ImportVoiceTnOrderResponse": {
+            "$ref": "#/components/schemas/ResponseStatus"
+          }
+        }
+      },
+      "RemoveImportedVoiceTnOrders": {
+        "type": "object",
+        "properties": {
+          "TotalCount": {
+            "type": "integer"
+          },
+          "Links": {
+            "$ref": "#/components/schemas/PaginationLinks"
+          },
+          "RemoveImportedVoiceTnOrderSummary": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/OrderSummary"
+            }
+          }
+        }
+      },
+      "RemoveImportedVoiceTnOrderRequest": {
+        "required": [
+          "TelephoneNumbers"
+        ],
+        "title": "RemoveImportedVoiceTnOrder",
+        "properties": {
+          "CustomerOrderId": {
+            "type": "string",
+            "description": "Internal customer order id for tracking the order. Only alphanumeric values, dashes and spaces are allowed",
+            "maxLength": 40
+          },
+          "TelephoneNumbers": {
+            "description": "Ten digit phone number with no dots or dashes. One or more is required. Use a PhoneNumber tag for each phone number in the list",
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "$ref": "#/components/schemas/TelephoneNumber"
+            }
+          }
+        }
+      },
+      "RemoveImportedVoiceTnOrderResponse": {
+        "type": "object",
+        "properties": {
+          "RemoveImportedVoiceTnOrder": {
+            "type": "object",
+            "properties": {
+              "CustomerOrderId": {
+                "type": "string"
+              },
+              "OrderCreateDate": {
+                "type": "string"
+              },
+              "AccountId": {
+                "type": "integer"
+              },
+              "CreatedByUser": {
+                "type": "string"
+              },
+              "OrderId": {
+                "type": "string"
+              },
+              "LastModifiedDate": {
+                "type": "string"
+              },
+              "ProcessingStatus": {
+                "type": "string",
+                "enum": [
+                  "RECEIVED",
+                  "PROCESSING",
+                  "COMPLETE",
+                  "PARTIAL",
+                  "FAILED"
+                ]
+              },
+              "TelephoneNumbers": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/components/schemas/TelephoneNumber"
+                }
+              },
+              "Errors": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/components/schemas/TelephoneNumberGroupError"
```

Comment:
Where would the requestor see problematic `tns` not being successfully imported? Inside `TelephoneNumberGroupError`?

Reply:
yes